pybinder/core.py

```diff
@@ -1562,6 +1566,22 @@ def is_operator(self):
             return self.spelling in PY_OPERATORS
         return False
 
+    @property
+    def is_getter_method(self):
+        """
+        Check if binder is a public method that returns a reference and the
+        class has an another public method public method named "Set<name>".
+
+        :return: *True* if getter, *False* otherwise.
+        :rtype: bool
+        """
+        if self.is_cxx_method and self.is_public and self.rtype.spelling.endswith("&"):
+            setter_name = f'Set{self.spelling}'
```

Comment:
Why just "Set*" methods? What's special about them vs. any method that returns a reference?

Reply:
It's an assumption that that convention (Value + SetValue) is essentially a property working with a internal member.